addons/sale_project/models/sale_order.py

```diff
@@ -171,8 +171,15 @@ def action_view_task(self):
 
     def action_create_project(self):
         self.ensure_one()
-        if not self.order_line:
-            return {'type': 'ir.actions.act_window_close'}
+        if not self.show_create_project_button:
+            return {
+                'type': 'ir.actions.client',
+                'tag': 'display_notification',
+                'params': {
+                    'type': 'danger',
+                    'message': _("This sales order is already linked to a project, or it does not involve any services."),
```

Comment:
```suggestion
                    'message': _("This Sales Order is either already linked to a project, or it does not involve any services."),
```
Make the capitalization consistent with other messages + slightly clearer formulation